nodeup/pkg/model/secrets.go

```diff
@@ -21,6 +21,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/pkg/errors"
```

Comment:
I think this would be a new import, and although it's handy, I don't think we want to add another dependency.  This does mean spelling out the error wrapping explicitly, I'm afraid.